### ISSUE TITLE
can't login when project on live server but not on my local

### ISSUE THREAD

**Issue by thulitha0109** [baijunyao/laravel-bjyblog#136]
![Screenshot from 2020-02-09 19-31-17](https://user-images.githubusercontent.com/57791655/74103555-fc54eb00-4b72-11ea-8ba0-ff1699e801f6.png)

I cloned your laravel project into my live server running Nginx server php7.2 mysql8 then i tried to login but this error occurred and i don't know what is the reason this only happing on live server but not in my localhost, the application works totally fine in my local environment. can anyone help me to get through this?

**Comment by baijunyao** [comment]
Maybe your server has some problem, every page is 404.
![image](https://user-images.githubusercontent.com/9360694/74104320-a1c98800-4b8e-11ea-8a7d-44651691175f.png)

**Comment by thulitha0109** [comment]
yeah maybe I'm using AWS EC2 instance and am new to it, im not using RDS database, i manually installed all php nginx mysql on EC2 instance, i have to look deeper onto my server thank you for respond.

**Comment by baijunyao** [comment]
You are welcome.
